exer3.rs

```diff
@@ -0,0 +1,84 @@
+fn main() {
+    //1. Convert miles to kilometers
+    println!("Enter the number of miles.");
+    let mut miles = String::new();
+    std::io::stdin().read_line(&mut miles).unwrap();
+    let number: f32 = miles.trim().parse().unwrap();
+    let kilo = number * 1.609;
+    miles.pop();
```

Comment:
Since you've already converted the string to a floating point number, it is unnecessary to modify the string here. It's easier to print `number` in Line 10 instead.